cmd/awat/import.go

```diff
@@ -29,44 +39,71 @@ var importCmd = &cobra.Command{
 	},
 }
 
-var getImportCmd = &cobra.Command{
-	Use:   "get",
-	Short: "Fetch an import by ID from the AWAT to get its status",
+var getImportByTranslationCmd = &cobra.Command{
+	Use:   "translation",
+	Short: "Get Imports by the Translation ID to which they correlate",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		imprt, _ := cmd.Flags().GetString(importID)
 		server, _ := cmd.Flags().GetString(serverFlag)
-		_ = model.NewClient(server)
-
-		installation, _ := cmd.Flags().GetString(installationID)
+		translation, _ := cmd.Flags().GetString(id)
 		awat := model.NewClient(server)
-		if (installation == "" && imprt == "") || (installation != "" && imprt != "") {
-			return errors.New("one and only one of translation-id or import-id must be specified")
+		if translation == "" {
+			return errors.New("Must provide an Translation ID to search by Translation ID")
```

Comment:
nit: error message has a circular dependency ;)
```suggestion
			return errors.New("must provide a Translation ID to search by Translation ID")
```

Reply:
whoops lmao